packages/data-transformer/src/normalizers/lowercase-normalizer.options.ts

```diff
@@ -0,0 +1,7 @@
+import {BaseNormalizerOptions} from "./base-normalizer.options";
+
+export class LowercaseNormalizerOptions extends BaseNormalizerOptions {
```

Comment:
I think we should move the options to normalizer-options because if we get a lot of normalizers it will get crowded fast